assets/queries/openAPI/parameter_object_undefined_type/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "46facedc-f243-4108-ab33-583b807d50b0",
+  "queryName": "Parameter Object With Undefined Type",
```

Comment:
The folder name is slightly different from the 'queryName' in the metadata file

Reply:
I think it's ok